docs/buildah-containers.md

```diff
@@ -24,6 +24,11 @@ Do not truncate IDs in output.
 
 Displays only the container IDs.
 
+**--all, -a**
+
+Also list information about containers which don't appear to be buildah working
```

Comment:
Changed the description to `List information about all containers, including those which were not created by and are not being used by buildah.`